instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/semconv/db/SqlClientAttributesGetter.java

```diff
@@ -22,7 +22,37 @@
  * OpenTelemetry specification.
  */
 public interface SqlClientAttributesGetter<REQUEST, RESPONSE>
-    extends DbClientCommonAttributesGetter<REQUEST, RESPONSE> {
+    extends DbClientAttributesGetter<REQUEST, RESPONSE> {
+
+  /**
+   * SqlClientAttributesExtractor will try to populate db.operation.name based on {@link
+   * #getRawQueryTexts(REQUEST)}, but this can be used to explicitly provide the operation name.
+   */
+  @Override
+  @Nullable
+  default String getDbOperationName(REQUEST request) {
+    return null;
+  }
+
+  /**
+   * SqlClientAttributesExtractor will try to populate db.operation.name based on {@link
+   * #getRawQueryTexts(REQUEST)}, but this can be used to explicitly provide the operation name.
+   */
+  @Override
+  @Nullable
+  default String getDbQueryText(REQUEST request) {
+    return null;
+  }
+
+  /**
+   * SqlClientAttributesExtractor will try to populate db.operation.name based on {@link
+   * #getRawQueryTexts(REQUEST)}, but this can be used to explicitly provide the operation name.
+   */
+  @Override
+  @Nullable
+  default String getDbQuerySummary(REQUEST request) {
+    return null;
+  }
```

Comment:
these methods have same defaults in super interface, but we have comments to remove the defaults in the super interface at which time we'll still want them here